Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -748,13 +763,12 @@
   - ENOUGH CABBAGES
   - FIVE HUNDRED AND NINETY-NINE US DOLLARS
   - FIVE TEENAGERS WITH ATTITUDE
-  - GODDAMN FUCKING PIECE OF SHIT ASSHOLE BITCH-CHRISTING CUNT-SMUGGLING SWEARING
+  - GODDAMN FUCKING PIECE OF SHIT ASSHOLE BITCH-CHRISTING SWEARING #  CUNT-SMUGGLING is ERP since cunt is pussy in slang, this whole thing should actually be removed but need more opinions
   - GREENTEXT
   - HERESY
   - HEROES IN A HALF SHELL
   - HIGH YIELD EXPLOSIVES
   - IMMORTALITY
-  - IT TO BE PAINTED BLACK
```

Comment:
this is a song referece

Reply:
well yeah, but like, painting black can mean other, much worse things

Reply:
There's still plenty of things in this file that can be misconstrued if you try hard enough. Don't remove the obvious song reference.

Reply:
> There's still plenty of things in this file that can be misconstrued if you try hard enough. Don't remove the obvious song reference.

It's not that good of a reference, and also im removing anything that can coorelate to specifically sexism and racism, since it was the main issue brought up in #26683, and this is just one of those things (also i don't get the reference)